java/kafka/producer/src/main/java/KafkaProducerExample.java

```diff
@@ -55,10 +55,9 @@ public static void main(String[] args) throws InterruptedException {
         }
 
         KafkaProducer<String, String> producer = new KafkaProducer<>(props);
-        log.info("Sending {} messages ...", config.getMessageCount());
 
         boolean blockProducer = System.getenv("BLOCKING_PRODUCER") != null;
-        boolean transactionalProducer = config.getAdditionalConfig().contains("transactional.id");
+        boolean transactionalProducer = System.getenv("KAFKA_TRANSACTIONAL_ID") != null;
         int msgPerTx = Integer.parseInt(System.getenv().getOrDefault("MESSAGES_PER_TRANSACTION", "10"));
```

Comment:
Same as above, should this be removed or `STRIMZI_MESSAGES_PER_TRANSACTION`? (plus documented in the README?)